### PR TITLE
Add further props to configure login page

### PR DESCRIPTION
**Pull request by letrungkien0210**
Hi @jakubknejzlik ,

Please review.

Thanks

### CODE REVIEW

File: src/login.js [hunk 1]
```diff
@@ -3,43 +3,91 @@ import { Checkbox } from "react-icheck";
...
+              </a>
+            );
+          case "google":
+            return (
+              <a href="#" className="btn btn-block btn-social btn-google btn-flat">Name
+                <i className="fa fa-google-plus"></i> Sign in using
+                Google+
+              </a>
+            );
+          default:
```
Comment (jakubknejzlik) [inline src/login.js]: Can we store this information in object elsewhere like:
```
let providers = {
  facebook: {title:'Sign in using Facebook'},
  google: {title:'Sign in using Google+'},
}
```

and iterate this project?

File: src/login.js [hunk 2]
```diff
@@ -3,43 +3,91 @@ import { Checkbox } from "react-icheck";
...
+          case "facebook":
+            return (
+              <a href="#" className="btn btn-block btn-social btn-facebook btn-flat">Name
+                <i className="fa fa-facebook"></i> Name
+                Sign in using Facebook
+              </a>
+            );
+          case "google":
+            return (
+              <a href="#" className="btn btn-block btn-social btn-google btn-flat">Name
```
Comment (jakubknejzlik) [inline src/login.js]: What is the `Name` text for? I don't see it in original html.

File: src/login.js [hunk 3]
```diff
@@ -3,43 +3,91 @@ import { Checkbox } from "react-icheck";
...
+      })
+    }
+    return authProviders;
+  }
+  
   render(){
+    let flagShowOR = true;
+    if(!this.props.userCredentialsEnabled || this.props.authProviders.length === 0){
+      flagShowOR = false;
+    }
```
Comment (jakubknejzlik) [inline src/login.js]: This can be also written as:
```
let flagShowOR = !this.props.userCredentialsEnabled || this.props.authProviders.length === 0
```

**Review by jakubknejzlik — CHANGES_REQUESTED** [review]

File: src/login.js (hunk 1 above)
Comment (letrungkien0210) [reply src/login.js]: Yes, sure.

File: src/login.js (hunk 3 above)
Comment (letrungkien0210) [reply src/login.js]: Thanks for your suggest.

File: src/login.js (hunk 2 above)
Comment (letrungkien0210) [reply src/login.js]: Sorry for my editor, sometimes move that to another place (Name for className property) which I don't see while committing codes.

File: src/login.js (hunk 1 above)
Comment (jakubknejzlik) [reply src/login.js]: Object is just to easily reference the value. You specify array of providers too. Let's say..

```
let providers = ['facebook','google']

let providersInfo  = {
  facebook: {title:'Sign in using Facebook'},
  google: {title:'Sign in using Google+'},
}

content = providers.forEach(providerCode => {
  let provider = providersInfo[providerCode]
  return (
    <a href="#" className="btn btn-block btn-social btn-google btn-flat">
       {providersInfo[provider].title}
    </a>
  )
})

```

just example :)

**Comment by letrungkien0210** [comment]
Hi @jakubknejzlik ,

I fixed your comments.

Please review.
Thanks.

File: src/login.js (hunk 2 above)
Comment (jakubknejzlik) [reply src/login.js]: Ok, still...can you please remove it?

Comment (letrungkien0210) [reply src/login.js]: removed it, please pull the latest code.

**Review by jakubknejzlik — APPROVED** [review]

**Pull request merged**